cvmfs/server/cvmfs_server_util.sh

```diff
@@ -976,10 +976,22 @@ _update_geodb() {
 
   # sanity checks
   [ -w "$dbdir"  ]   || { echo "Directory '$dbdir' doesn't exist or is not writable by $(whoami)" >&2; return 1; }
-  [ ! -f "$dbfile" ] || [ -w "$dbfile" ] || { echo "GeoIP database '$dbfile' is not writable by $(whoami)" >&2; return 2; }
 
   # check if an update/installation needs to be done
-  if [ ! -f "$dbfile" ]; then
+  if [ -n "$CVMFS_GEO_DB_FILE" ]; then
```

Comment:
Maybe

```
if [ x"$CVMFS_GEO_DB_FILE" != x"" ]
```

Reply:
No, I think that's worse style and just reflects a lack of knowledge of both test -n and the fact that double quotes makes it unnecessary to use the double-x pattern.  That pattern is only necessary when not using quotes.

Reply:
I was under the impression that it was needed to comply with old/weird shell version. I would keep it just for consistency with the whole codebase at this point.

Reply:
There are other uses of -n in cvmfs_server.